pool/tick_bitmap.gno

```diff
@@ -31,11 +30,9 @@ func (pool *Pool) tickBitmapFlipTick(
 	}
 
 	wordPos, bitPos := tickBitmapPosition(tick / tickSpacing)
+	pool.tickBitmaps[wordPos] = pool.tickBitmaps[wordPos].NilToZero()
```

Comment:
I see `tickBitmaps` being used multiple times. Abstract it into a helper func?

Reply:
22d47515d930c650b5e6f43b8351b39bea1eb613